### ISSUE TITLE
[Bug Report] Carousel items with fade-transition blinks instead of fading one into another.

### ISSUE THREAD

**Issue by Magiczne** [vuetifyjs/vuetify#10809]
### Environment
**Vuetify Version:** 2.2.15
**Vue Version:** 2.6.11
**Browsers:** Chrome 80.0.3987.132
**OS:** Windows 10

### Steps to reproduce
Create carousel with fade transition

### Expected Behavior
Slides should be fading smoothly one into another (one fades out, another fades in)

### Actual Behavior
First cycle - everything is working correctly. Images are smoothly fading one into another.
Next cycles - image is fading out then next image blinks into slider.

### Reproduction Link
<a href="https://codepen.io/Magiczne/pen/xxGYxyZ" target="_blank">https://codepen.io/Magiczne/pen/xxGYxyZ</a>

**Comment by anisabboud** [comment]
In ran into the same issue. Carousel fade-transition isn't smooth.

You can also see this issue in the official Carousel with **Custom transition** example: https://vuetifyjs.com/en/components/carousels/#custom-transition

In that example:
- If you are on a some slide `x` and you go _forward_ (using the arrows/delimiters) to any upcoming slide `> x`, you see a blank white screen for a second before the upcoming slide fades in.
- However, if you are on some slide `x` and you go _backwards_ (using the arrows/delimiters) to any previous slide `< x`, the transition is fast and works well!

![vuetify-carousel-fade-transition](https://user-images.githubusercontent.com/5672108/78837548-fd808800-79fc-11ea-911c-be33911692f9.gif)

So perhaps the problem is only with transition but not with reverse-transition...

**Comment by mikecbone** [comment]
Seems to be a loading issue for me where it flickers on the first round, and then seems smooth on the second run through. This isn't ideal and trying the `eager` prop doesn't fix it either.

**Comment by Magiczne** [comment]
I'm currently using vuetify version **2.2.27**, and the behavior is slightly different.

Now it behaves like:
Let's assume we have `n` items in carousel.
- If you're on the slide `x` where `x < n` and you go **forward** you will see a background blink before next slide is showing up.
- If you're on the slide `x` where `x = n` and you go **forward** transition fires up correctly and last slide fades into first slide correctly.
- If you're on the slide `x` where `x != 1` and you go **backwards** you will see a background blink
- If you're on the slide `x` where `x = 1` and you go **backward** transition fires up correctly

**Comment by Magiczne** [comment]
Additionaly using basic v-window code like below
```
<v-window-item v-for="slide in slides" 
                            transition="fade-transition"
                            reverse-transition="fade-transition"
                            :key="slide">
    <v-img eager :src="slide">
</v-window-item>
```
results in behavior described by @anisabboud.

**Comment by anisabboud** [comment]
@Magiczne we are describing the same problem/behavior in different words.
When you are on slide `x = n` and go forward, it's like going backwards because it scrolls all the way back to 1.

In the meantime, I worked around this issue using some css:
```css
.v-carousel .v-window-item {
  position: absolute;
  top: 0;
  width: 100%;
}
```
(I noticed that for some reason the next slide was located at the bottom before fading in, which is why we aren't seeing the fade-in and seeing blank during the transition. This code positions it back up so that it's visible while fading.)

**Comment by itsgifnotjiff** [comment]
> @Magiczne we are describing the same problem/behavior in different words.
> When you are on slide `x = n` and go forward, it's like going backwards because it scrolls all the way back to 1.
> 
> In the meantime, I worked around this issue using some css:
> 
> ```css
> .v-carousel .v-window-item {
>   position: absolute;
>   top: 0;
>   width: 100%;
> }
> ```
> 
> (I noticed that for some reason the next slide was located at the bottom before fading in, which is why we aren't seeing the fade-in and seeing blank during the transition. This code positions it back up so that it's visible while fading. It also fixes #10780 for me.)

The height of the carousel can not be set to auto in order for your workaround to work. I am trying t find out why.

**Comment by jckling** [comment]
I'm using v-carousel in v-card, remove auto height fixed the problem and become ugly. 😢
